tracing/README.md

```diff
@@ -69,13 +70,17 @@ _n_ new items were taken from a queue, and so on.
 to indicate that some code takes place within the context of that `Span`:
 
 ```rust
+# #[macro_use] extern crate tracing;
+# use tracing::Level;
```

Comment:
I'd rather any `use` statements necessary for the examples to work just be part of the example, so we don't need to comment these out.